packages/cursorless-engine/src/test/generateSpokenForm.test.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Command,
+  TestCaseFixtureLegacy,
+  getRecordedTestPaths,
+  getRecordedTestsDirPath,
+  serialize,
+  shouldUpdateFixtures,
+} from "@cursorless/common";
+import * as yaml from "js-yaml";
+import * as assert from "node:assert";
+import * as fs from "node:fs";
+import { promises as fsp } from "node:fs";
+import * as path from "node:path";
+import { canonicalizeAndValidateCommand } from "../core/commandVersionUpgrades/canonicalizeAndValidateCommand";
+import { generateSpokenForm } from "../core/commandVersionUpgrades/generateSpokenForm";
+
+suite("Generate spoken forms", () => {
+  const relativeDir = path.dirname(getRecordedTestsDirPath());
+
+  getRecordedTestPaths().forEach((testPath) =>
+    test(path.relative(relativeDir, testPath.split(".")[0]), () =>
+      runTest(testPath),
+    ),
+  );
+});
+
+async function runTest(file: string) {
+  const buffer = await fsp.readFile(file);
+  const fixture = yaml.load(buffer.toString()) as TestCaseFixtureLegacy;
+  const spokenForm = await generateSpokenFormWithHatTokenMapSuffix(
+    file,
+    fixture.command,
+  );
+
+  if (spokenForm == null) {
+    return null;
+  }
+
+  if (shouldUpdateFixtures()) {
```

Comment:
Do we have a launch config to actually run this?

Reply:
Same one as before.

Reply:
Ah right.  Was looking for way to update just unit tests.  Prob fine to just use full test update